### PR TITLE
Added a toxicity detection script for the future toxicity detection example

### PR DESCRIPTION
**Pull request by Etesam913**
## Summary
Added a run_task.py script in the toxicity_detection folder. The script accepts an input string for toxicity detection checking. Toxicity is calculated using the [detoxify](https://github.com/unitaryai/detoxify) library. 

detoxify will have to be installed to use this script `pip install detoxify` .

Created some of the base files for the example that will be built later. These base files were copied from the mnist example.

## Testing

https://user-images.githubusercontent.com/55665282/170797844-2b5d18f0-d29c-40e4-a6d7-563fdc235972.mov

From the video it is clear that the script can be run with an input string. The toxicity of the input string (as calculated by detoxify) is outputted.

### CODE REVIEW

**Comment by Etesam913** [comment]
What's the formatter to pass the code-style test for python files. Prettier works well for jsx files.

**Comment by pringshia** [comment]
> What's the formatter needed to pass the code-style test for python files? Prettier works for jsx files.

we are using `black`

File: examples/remote_procedure/toxicity_detection/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Mephisto MNIST Model Evaluator demo
+
+This task is a simple model evaluation demo, presenting the worker with three spaces to draw images in, running an MNIST model on them to predict an output, and then allowing the worker to assess and correct that output.
+
+Introduced in [#677](https://github.com/facebookresearch/Mephisto/pull/677), some screenshots and discussion there.
```
Comment (pringshia) [inline examples/remote_procedure/toxicity_detection/README.md:5]: Stale README from the copy-paste

File: examples/remote_procedure/toxicity_detection/hydra_configs/conf/launch_with_local.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  blueprint:
+    task_source: ${task_dir}/webapp/build/bundle.js
+    link_task_source: false
+    # NOTE pick something based on your task
+    block_qualification: test_qual_block
+    units_per_assignment: 1
+  task:
+    allowed_concurrent: 1
+    task_name: remote-procedure-test-task
+    task_title: "Provide feedback on our MNIST model"
```
Comment (pringshia) [inline examples/remote_procedure/toxicity_detection/hydra_configs/conf/launch_with_local.yaml:16]: stale metadata

File: examples/remote_procedure/toxicity_detection/hydra_configs/conf/launch_with_local.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    # NOTE you'll want to update your task description
+    task_description: "You will draw digits. Try to fool our MNIST model, and provide us the correct label."
+    # NOTE set a reasonable reward
+    task_reward: 0.05
+    # NOTE will want real tags
+    task_tags: "mnist,drawing,models,correction"
+    # We expect to handle 25 people using the MNIST model at once
```
Comment (pringshia) [inline examples/remote_procedure/toxicity_detection/hydra_configs/conf/launch_with_local.yaml:23]: we won't be using an mnist model for this one so comments will need to be updated

File: examples/remote_procedure/toxicity_detection/model.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Using the implementation provided at
+# https://github.com/aaron-xichen/pytorch-playground/blob/master/mnist/model.py
```
Comment (pringshia) [inline examples/remote_procedure/toxicity_detection/model.py:8]: you can remove this comment

**Pull request closed without merging**